docs/products/kafka/kafka-connect/howto/splunk-sink.rst

```diff
@@ -0,0 +1,116 @@
+
+    The ``key.converter`` and ``value.converter`` sections define how the topic messages will be parsed and need to be included in the connector configuration.
+
+    When using Avro as source data format, you need to set the following parameters
+
+    * ``value.converter.schema.registry.url``: pointing to the Aiven for Apache Kafka schema registry URL in the form of ``https://APACHE_KAFKA_HOST:SCHEMA_REGISTRY_PORT`` with the ``APACHE_KAFKA_HOST`` and ``SCHEMA_REGISTRY_PORT`` parameters :ref:`retrieved in the previous step <connect_splunk_sink_prereq>`.
+    * ``value.converter.basic.auth.credentials.source``: to the value ``USER_INFO``, since you're going to login to the schema registry using username and password.
+    * ``value.converter.schema.registry.basic.auth.user.info``: passing the required schema registry credentials in the form of ``SCHEMA_REGISTRY_USER:SCHEMA_REGISTRY_PASSWORD`` with the ``SCHEMA_REGISTRY_USER`` and ``SCHEMA_REGISTRY_PASSWORD`` parameters :ref:`retrieved in the previous step <connect_splunk_sink_prereq>`. 
+
+
+Create a Kafka Connect connector with the Aiven Console
+'''''''''''''''''''''''''''''''''''''''''''''''''''''''
+
+To create the connector, access the `Aiven Console <https://console.aiven.io/>`_ and select the Aiven for Apache Kafka® or Aiven for Apache Kafka Connect® service where the connector needs to be defined, then:
+
+1. Click on the **Connectors** tab
+2. Clink on **Create New Connector**. This button is only enabled for services :doc:`with Kafka Connect enabled <enable-connect>`.
+3. Select the **Splunk sink**
+4. Under the *Common* tab, locate the **Connector configuration** text box and click on **Edit**
+5. Paste the connector configuration (stored in the ``splunk_sink.json`` file) in the form
+6. Click on **Apply**
+
+.. Note::
+
+    The Aiven Console parses the configuration file and fills the relevant UI fields. You can review the UI fields across the various tabs and change them if necessary. The changes will be reflected in JSON format in the **Connector configuration** text box.
+
+7. After all the settings are correctly configured, click on **Create new connector**
+8. Verify the connector status under the **Connectors** tab
+9. Verify the data in the target Splunk instance
+
+.. Note::
+
+    Connectors can also be created using the dedicated :ref:`Aiven CLI command <avn_service_connector_create>`.
+
+Example: Create an Splunk sink connector with a server having no authorization
+----------------------------------------------------------------------------
+
+If you have a topic named ``data_logs`` that you want to sink to a Splunk server in the ``kafka_logs`` index:
+
+.. code-block:: json
+
+    {
+        "name":"data_logs_splunk_sink",
+        "connector.class": "com.splunk.kafka.connect.SplunkSinkConnector",
+        "splunk.hec.token": "SPLUNK_HEC_TOKEN",
+        "splunk.hec.uri": "SPLUNK_HEC_URI",
+        "splunk.indexes": "kafka_logs",
+        "topics": "data_logs"
+    }
```

Comment:
Why is it having no authorization? I thought the token was sent with the request.

In my case, Splunk gave me a self-signed https endpoint, so I also needed `"splunk.hec.ssl.validate.certs": "false",` in my config (kind of just putting this here because I don't have a working version yet so it could be useful later)